src/main/java/core/basesyntax/service/impl/ReportWriterImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportWriterService;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ReportWriterImpl implements ReportWriterService {
+    private List<String[]> report = new ArrayList<>();
+
+    public List<String[]> getReport() {
+        writeHead();
+        writeBody(Storage.getAll());
+        return report;
+    }
+
+    public void writeHead() {
```

Comment:
```suggestion
    private void writeHead() {
```

I think we could make this method private